aries-site/src/pages/foundation/tshirt-sizing.mdx

```diff
@@ -47,17 +51,74 @@ xxlarge: `${baseSpacing * 64}px`, // 1536px
 full: '100%',
 ```
 
+### Evolution to v7: The "Dual Doubling" Scale
+
+With the introduction of HPE theme v7, the t-shirt sizing scale has been significantly expanded using a "Dual Doubling" approach to provide more granular sizing options. This new scale addresses gaps in the previous scale and offers greater design flexibility.
+
+**Key highlights of the Dual Doubling scale:**
+
+- **Introduces in-between steps** which address gaps in the current scale, making desired design layouts easier to execute
+- **Has parity with existing scale** which means that all designs may be recreated with the new scale without any layout shifts
+- **New `medium` = current `medium`**; however, rest of t-shirt sizes will need to be mapped to achieve design parity
+- **Every t-shirt size may be composed** by smaller t-shirt sizes and/or divided into halves, thirds, quarters, ... 12ths, etc. (e.g. `small` + `small` = `large`, or `xsmall` + `xsmall` + `xsmall` = `large`, or even `medium` + `xsmall` = `large`)
+- **Scale consists of a primary scale** based on the doubling of a 24px base unit (same as v6) + a supplementary scale doubling a 16px base unit
+
+**Results in the following pixel progression:**
+1, 2, 3, 4, 6, 8, 12, 16, 24, 32, 48, 64, 96, 128, 192, 256, 384, 512, 768, 1024, 1536, 2048
+
+Below are the sizing values from the [HPE theme v7](https://github.com/grommet/grommet-theme-hpe/blob/master/CHANGELOG.md#700) and above which implements this expanded scale:
+
+### Container sizes comparison (width, height, size, columns, rows)
```

Comment:
Soft opinion but I feel like the notes don't feel like they add much value and create more visual clutter. Open to other opinions